students/rod_musser/lesson04/mailroom.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+import sys
+
+donors = {
+    'Carmelo Anthony': [1, 50],
+    'Damien Lillard': [100.50, 99, 10000],
+    'CJ McCollum': [24000, 70, 100, 5, 300],
+    'Hassan Whiteside': [10000],
+    'Terry Stotts': [500, 500, 100, 100, 100],
+}
+
+
+welcome_prompt = "\n".join(("Welcome to the Local Charity Mail Room System",
+                  "Please choose from the following options:",
+                  "1 - Send a Thank You",
+                  "2 - Create a Report",
+                  "3 - Quit",
+                  ">>> "))
+
+
+def main():
+    while True:
+        response = input(welcome_prompt)
+        menu_selection = main_menu_dict.get(response)
+        if menu_selection is None:
+            print("Not a valid option!")
+        else:
+            menu_selection()
+
+
+def list_donors():
+    newLine = '\n'
```

Comment:
```suggestion
    new_line = '\n'
```

should be snake_case